Form-Controls/index.html

```diff
@@ -16,12 +16,57 @@ <h1>Product Pick</h1>
   <form>
     <!-- write your html here-->
     <!-- try writing out the requirements first-->
+     <!--                        Requirements:
+     1) What is the customer's name? I must collect this data, and validate it. 
+     But what is a valid name? I must decide something.
+     2) What is the customer's email? I must make sure the email is valid. 
+     Email addresses have a consistent pattern.
+     3) What colour should this t-shirt be? I must give 3 options. How will I make sure they don't pick other colours?
+     4) What size does the customer want? I must give the following 6 options: XS, S, M, L, XL, XXL
+     5) When do they want the t-shirt to be delivered? I must collect a date and make sure that date is in the next four weeks. 
+     How will I do this? How will I make sure there are no mistakes about the date?-->
+     
+      <!-- Customer Name -->
+      <label for="customer-name">Name:</label>
+      <input type="text" id="customer-name" name="customer-name" required pattern=".{2,}" title="Name must be at least 2 characters long">
```

Comment:
nice use of `pattern` here!

Reply:
Hi @jamesbaskerville -- thanks for the feedback and guidance!

For Lighthouse Audit: I'll run a Lighthouse check for Accessibility and Performance to bring the scores closer to 100. I’ll also make some adjustments to improve the visual engagement of the site.

I appreciate the note on using the pattern attribute! I’ll ensure all form fields have appropriate patterns and titles for validation consistency.